test/Microsoft.OpenApi.Tests/Workspaces/OpenApiWorkspaceTests.cs

```diff
@@ -31,54 +30,8 @@ public void OpenApiWorkspacesAllowDocumentsToReferenceEachOther()
         {
             var workspace = new OpenApiWorkspace();
 
-            workspace.AddDocument("root", new OpenApiDocument
-            {
-                Paths = new OpenApiPaths
-                {
-                    ["/"] = new OpenApiPathItem
-                    {
-                        Operations  = new Dictionary<OperationType, OpenApiOperation>
-                        {
-                            [OperationType.Get] = new OpenApiOperation
-                            {
-                                Responses = new OpenApiResponses
-                                {
-                                    ["200"] = new OpenApiResponse
-                                    {
-                                       Content = new Dictionary<string,OpenApiMediaType>
-                                       {
-                                           ["application/json"] = new OpenApiMediaType
-                                           {
-                                               Schema = new OpenApiSchema
-                                               {
-                                                   Reference = new OpenApiReference
-                                                   {
-                                                       Id = "test",
-                                                       Type = ReferenceType.Schema
-                                                   }
-                                               }
-                                           }
-                                       }
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            });
-            workspace.AddDocument("common", new OpenApiDocument
-            {
-                Components = new OpenApiComponents
-                {
-                    Schemas = {
-                        ["test"] = new OpenApiSchema
-                        {
-                            Type = "string",
-                            Description = "The referenced one"
-                        }
-                    }
-                }
-            });
+            workspace.AddDocument("root", new OpenApiDocument {Paths = new OpenApiPaths {["/"] = new OpenApiPathItem {Operations = new Dictionary<OperationType, OpenApiOperation> {[OperationType.Get] = new OpenApiOperation {Responses = new OpenApiResponses {["200"] = new OpenApiResponse {Content = new Dictionary<string, OpenApiMediaType> {["application/json"] = new OpenApiMediaType {Schema = new OpenApiSchema {Reference = new OpenApiReference {Id = "test", Type = ReferenceType.Schema}}}}}}}}}}});
```

Comment:
I don't believe this change represents an improvement. Could you revert please?

Reply:
i dont know what happened there. fixed